omero/sysadmins/admins-with-restricted-privileges.txt

```diff
@@ -0,0 +1,418 @@
+    Inside the groups the restricted admin is a member of, he/she
+    has the same privileges as other group members of that group
+    additionally to his/her adminstrative privileges.
+
+.. note::
+    **Deleting privileges**: :term:`Sudo` privilege has implicitly
+    in itself a Delete power over the data the administrator is
+    sudoed in as. If you want to prevent restricted admin from
+    Deleting of others' data entirely, do not give :term:`Delete Data`
+    AND do not give :term:`Sudo` privileges.
+
+.. note::
+    **Privilege escalation**: Administrator with restricted privileges
+    (restricted admin) are prevented from escalation of their privileges.
+    Escalation cannot be achieved by them neither by sudoing as full
+    administrator or higher-privileged restricted admin, nor by creating
+    a full administraotor or new restricted admin with privileges higher
+    than the creating restricted admin. A restricted admin can use
+    their :term:`Sudo` privilege, the prevention of escalation means
+    they will be able to execute maximally the privileges level they
+    were enjoying before sudoing, no more privileges can be acquired
+    by the sudoing itself. In case the sudoed-as user has less privileges
+    than the sudoing restricted admin, these lesser privileges will apply
+    also for the restricted admin as long as sudoed-in.
+
+.. _Workflow 1:
+
+Workflow 1: Data viewer
+-----------------------
+Even if you do not give any explicit privileges to the administrator with
+restricted privileges, this new administrator has some useful powers.
+These include browsing and viewing all the data of all users in all
+groups (including the groups where he/she is not a member).
+The administrator with restricted privileges
+is also able to Download all the data in all types of groups.
+Further, he/she can view user and group information, such as usernames,
+e-mail addresses, group permission levels and lists of all users
+and groups. He/she is not able to annotate, edit or delete any of the
+data or change any user or group informations though. Note that any
+administrator with restricted privileges described below or otherwise
+created combining the privileges at will would be able to perform the
+Data viewer workflow as well.
+
+Workflow 1: Data viewer in clients:
+
+- OMERO.insight: is not designed to show any groups or data you are not
+  a member of. The Data Viewer workflow is preferably
+  executed using OMERO.web or CLI
+
+- OMERO.web: viewing and downloading the data,
+  see :help:`Viewing Data <viewing-data>`
+
+- CLI: listing all images, groups and users, download::
+
+    # List all users on server
+    $ bin/omero user list
+    # List all groups on server
+    $ bin/omero group list
+    # List all images on server
+    $ bin/omero fs images
+
+.. _Workflow 2:
+
+Workflow 2: Importer
+--------------------
+The Importer's task is to import images into OMERO for other users,
```

Comment:
I had some confusion here between "Importer" (the tool/process) and "Importer" (the user having privs to import).  Maybe clarify with "Importer role"; same for the other workflows.